### PR TITLE
Update empty mailboxes: Icons and Texts

### PR DESCRIPTION
**Pull request by JulianAdams4**

### CODE REVIEW

File: email_mailbox/src/lang/es.json [hunk 1]
```diff
@@ -66,17 +66,41 @@
...
-        "subheader": "Bien!"
+        "header": "No tienes Spam",
+        "subheader": "¡Qué bien! Mantengamoslo así"
+      },
+      "allmail": {
+        "header": "No hay Correos",
+        "subheader": "¡En lo absoluto!"
+      },
+      "search": {
+        "header": "No search results",
```
Comment (Hirobreak) [inline email_mailbox/src/lang/es.json]: THIS IS NOT SPANISH

File: email_mailbox/src/lang/es.json [hunk 2]
```diff
@@ -66,17 +66,41 @@
...
+        "subheader": "Trash & Spam emails are not displayed"
       },
       "default": {
-        "header": "No tienes correos todavía",
-        "subheader": "Envía un correo para comenzar"
+        "header": "You have no emails yet",
```
Comment (Hirobreak) [inline email_mailbox/src/lang/es.json]: THIS IS NOT SPANISH! WHY REMOVE SPANISH TRANSLATIONS?!

File: email_mailbox/src/lang/es.json (hunk 1 above)
Comment (JulianAdams4) [reply email_mailbox/src/lang/es.json]: Read with spanish accent. You're welcome.

File: email_mailbox/src/lang/es.json (hunk 2 above)
Comment (javierron) [reply email_mailbox/src/lang/es.json]: OMG

**Pull request merged**